recipes/tranquilizer/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2018, Anaconda, Inc.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
This file shouldn't be needed as it's already included in the source tarball.

Reply:
done